ace.d.ts

```diff
@@ -273,6 +273,24 @@ export namespace Ace {
     type: string;
   }
 
+export interface TooltipMarker {
+  range: Range;
+  tooltipText: string;
```

Comment:
Do we have tooltiptext in lsp or do we need to create a dom node from markdown?

Reply:
If we consider [LSP spec for diagnostics](https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#diagnostic), which would be the best fit for this functionality, it only has `message: string`. Having only text would be sufficient for this case. We could also provide a way to pass HTML, but that could be an extension implemented later.